parsers/markdown.c

```diff
@@ -58,18 +59,25 @@ static kindDefinition MarkdownKinds[] = {
 	{ true, 'T', "l4subsection",  "level 4 sections" },
 	{ true, 'u', "l5subsection",  "level 5 sections" },
 	{ true, 'n', "footnote",      "footnotes" },
+	{ true, 'h', "hashtag",       "hashtag"},
 };
 
-static fieldDefinition MarkdownFields [] = {
+static fieldDefinition MarkdownFields[] = {
 	{
-	  .enabled     = false,
-	  .name        = "sectionMarker",
-	  .description = "character used for declaring section(#, ##, =, or -)",
-	},
```

Comment:
It seems that the original code uses tab characters for indentation. Let's do the same in this pull request.

Reply:
Now changed in the same way.